sources/typesafe-config/src/main/java/io/smallrye/configsource/TypeSafeConfigConfigSource.java

```diff
@@ -0,0 +1,53 @@
+package io.smallrye.configsource;
+
+import java.io.Serializable;
+import java.util.Collections;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import org.eclipse.microprofile.config.spi.ConfigSource;
+
+import com.typesafe.config.Config;
+import com.typesafe.config.ConfigFactory;
+
+public class TypeSafeConfigConfigSource implements ConfigSource, Serializable {
+
+    static final int DEFAULT_ORDINAL = 50;
```

Comment:
Any particular reason to set the default ordinal to 50?

Reply:
I've to say it's arbitrary. I just thought it should be lower than 100. If you've a better suggestion, no problem. In any case it can be overwritten in the config.

Reply:
After #174 you can also extend EnabledConfigSource (that extends SelfConfiguringConfigSource) where you get a way to enable/disable the source and to configure the Ordinal.

Reply:
@phillip-kruger DO you suggest we should wait for #174 to be merged before merging this PR with the `EnabledConfigSource` in use?

Reply:
No, we can always add that later. Also #174 is not approved yet and that might take a while